Data/future/cbot/margins/EH.csv

```diff
@@ -1,61 +1,193 @@
 date,initial,maintenance
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-019.html
 20120117,4725,3500
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-221.html
 20120529,4050,3000
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv12-328.html
 20120806,4455,3300
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-168.html
+20130102,3630,3300
+20130212,3080,2800
 20130409,5130,3800
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv13-594.html
+20130410,4180,3800
+20131202,2970,2700
+20131204,4180,3800
+20140102,2970,2700
 20140103,5940,4400
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-114.html
+20140106,4840,4400
+20140203,2970,2700
+20140205,4840,4400
+20140303,2970,2700
+20140305,4840,4400
 20140321,6480,4800
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-138.html
+20140324,5280,4800
+20140402,3300,3000
+20140403,5280,4800
 20140404,8370,6200
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-204.html
-20140527,6820,6200
-# https://www.cmegroup.com/tools-information/lookups/advisories/clearing/Chadv14-279.html
+20140407,6820,6200
+20140502,3850,3500
+20140505,6820,6200
+20140602,3850,3500
+20140604,6820,6200
+20140702,3850,3500
+20140703,6820,6200
```

Comment:
Same here, looks like the end of one contract and the start of another. Its choppy churn and not sure its realistic.

i.e. more a byproduct of us only selecting the margins for the front-contract. If so we need to select say mid-month margins